docs/ARCHITECTURE.md

```diff
@@ -1,25 +1,33 @@
 # Architecture
-The BO1LER, a fully customisable design system (DS), is developed using the [Lit](https://lit.dev/docs/) template and Figma. The design system serves as a core unit for UI components, styles, and guidelines, ensuring consistency and efficiency across our applications.
+
```

Comment:
Please revert back to the previous state where there was no new line added after every title. If you are using VS-Code then the new line might have been added by the prettier code formatter extension. Please disable that and remove the newly added line and save the file. Later on after committing your changes, you can re-enable the extension.

Reply:
done